src/consensus/params.h

```diff
@@ -13,6 +13,15 @@
 
 namespace Consensus {
 
+enum BuriedDeploymentPos
+{
+    DEPLOYMENT_BIP16,
```

Comment:
Note that the title says BIP90, but BIP90 never mentions BIP16. So either the code needs to change or the title.

Reply:
Couldn't bip90 change as it does with new additions like hopefully csv and segwit?

Reply:
No, I'd rather not change BIPs to be a running target. The previous release implements BIP 90, so you can't change it after it is published.